extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -444,6 +444,13 @@ public void setSource(Source source) {
         @ConfigItem
         public Map<String, String> extraParams;
 
+        /**
+         * If enabled the state, session and post logout cookies will have their 'secure' parameter set to 'true'.
```

Comment:
@mqs24d Can you please add after `set to 'true'` something like this:
```
... when HTTP is used. It may be necessary when running behind an SSL terminating reverse proxy.
```
When I first looked at the new property I got a feeling that some users may think this property must be set to `true` always, so some extra clarifications might help...
Oh yes, and please add a getter and setter - while keeping the property as public - as I said, we tend to access properties directly from `quarkus-oidc`, but for `TenantConfigResolver` users it is handy to have the methods as well :-)
Thanks

Reply:
@sberyozkin incorporated your proposal... Sorry for that stupid noob mistakes. I still fighting with Intellij getting the workspace set up properly.

Reply:
@mqs24d  - cool, thanks for a fast update, all you updates have been perfect really :-)